packages/microcosm-www-next/src/components/graphic3.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Graphic3 = ({ fill }) => (
+  <figure>
+    <img
+      src="http://via.placeholder.com/600x600"
+      alt="TODO"
+      className="section__graphic__figure"
+      data-module="ObserveGraphic"
+      data-section="3"
+    />
+  </figure>
+)
+
+export default Graphic3
```

Comment:
Are these graphics more or less the same conceptually? What you had a single `Graphic` component that you passed different props into?

Reply:
yes, will most likely be the same so I will refactor that